performance/models.py

```diff
@@ -7,18 +7,19 @@
 
 def eqprs_file_path(instance, filename):
     return (
-        f"eqprs_uploads/{uuid.uuid4()}/{filename}"
+        # f"eqprs_uploads/{uuid.uuid4()}/{filename}"
+        f"eqprs_uploads/{filename}"
     )
 
 
 class EQPRSFileUpload(models.Model):
-    user = models.ForeignKey(User, models.DO_NOTHING)
-    task = models.ForeignKey(Task, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, models.DO_NOTHING, default=0)
+    task = models.ForeignKey(Task, on_delete=models.CASCADE, null=True, blank=True)
```

Comment:
why are you allowing blanks and nulls here? Can't we create the task on save and thus meet the db requirements?

Reply:
while creating the Task I need the object's id to set `source_id` of `Indicator`. So, first I need to save the object, then I can create the Task. That's why I allowed it to be null & blank.